inc/alert.class.php

```diff
@@ -416,7 +416,7 @@ public function showForm($ID, $options = []) {
 
    static function displayOnCentral() {
       echo "<tr><th colspan='2'>";
-      self::displayAlerts(['show_only_helpdesk_alerts' => Session::getCurrentInterface() == 'helpdesk']);
+      self::displayAlerts(['show_only_helpdesk_alerts' => Session::getCurrentInterface() == 'central']);
```

Comment:
`show_only_helpdesk_alerts` is used to display alerts related to `tickets`. It has nothing to do with the `central`/`helpdesk` interface state.
```suggestion
      self::displayAlerts();
```

Reply:
the goal of the flag is to display on the helpdesk page (where's ticket are present) because on self-service, you can make the tickets page as index.

Having alerts displayed on technician ticket's page is interesting but not the goal on the flag.

As far i remember, Central page (for technician) always displays alerts, there is no flag (yet) for this specific page

Reply:
revert code